bld/namelist_files/namelist_defaults_ctsm.xml

```diff
@@ -1021,6 +1025,8 @@ p
 >lnd/clm2/initdata_map/clmi.FHISTSp.2013-01-01.ne0CONUSne30x8_mt12_simyr2013_c200806.nc
 </finidat>
 
+<!-- NEON SITE DATA - this should not be needed once the site data is established -->
```

Comment:
It should already find an IC file that's as close as possible, so this should neither need or should be set.

Reply:
For a NEON site the initial conditions file (if used) would come from a previously spun up NEON case.  For a generic single point run, however, users may want to point to initial conditions from a global case?    Would this functionality be possible here?

Reply:
The IC file for each NEON site could be put in the user-mod directory for it. Or if you need to have more than one file matched per NEON site (depending on the starting date or compset) you could add it here into the namelist defaults file.

For the generic case the IC file matching will already use a suitable global case to interpolate from. So your request is already granted. That's why I suggest removing this here until you have specific IC files for NEON sites. Then we should decide how it should be done (user-mod directories or in the namelist_defaults), but we should wait until those are ready.